website/addons/dropbox/utils.py

```diff
@@ -79,7 +79,12 @@ def log(self, action, extra=None, save=False):
 
 
 def is_subdir(path, directory):
-    if not (path and directory):
+    if not path:
+        return False
+    # directory is root directory
+    if directory == '' or directory == '/':
```

Comment:
Why check for the empty string here? I think only `'/'` should be considered the root.